welcome-to-sourcery.py

```diff
@@ -18,17 +16,125 @@ def refactoring_example(spellbook):
             result.append(spell)
     return result
```

Comment:
This example already contains 2 refactorings:

* convert for loop into list comprehension
* inline variable that is immediately returned

Perhaps, we could change `return result` to `print(result)`, so that Sourcery suggests only 1 refactoring for this function.